crates/runestick/src/modules/int.rs

```diff
@@ -12,6 +12,7 @@ pub fn module() -> Result<Module, ContextError> {
     module.function(&["parse"], parse)?;
     module.function(&["max"], i64::max)?;
     module.function(&["min"], i64::min)?;
+    module.function(&["abs"], i64::abs)?;
```

Comment:
Huh, I think this is already added as an associated fn, which should mean it's registered under this name already.